
particle_filter:
  use_gpu: true 

  cpu:
    sample_count: 100

  gpu:
    sample_count: 2000

    # set true to use custom shader definitions file defined below
    # if set to false, the default shader will be used
    use_custom_shaders: false

    vertex_shader_file: /path/to/custom/vertex_shader.vertexshader
    fragment_shader_file: /path/to/custom/fragment_shader.fragmentshader

    # optional (values: none | /path/to/shader)
    geometry_shader_file: none

  # State moving average update rate in [0.0; 1.0]
  #   1: unfiltered wiggly estimates.
  #   0: constant
  moving_average_update_rate: 1.0

  max_kl_divergence: 2.0

  observation:
    occlusion:
      initial_occlusion_prob: 0.1 #0.1
      p_occluded_visible: 0.1 #0.1
      p_occluded_occluded: 0.7 #0.7
    kinect:
      tail_weight: 0.01 #0.01, 0.998
      model_sigma: 0.003 #0.003, 0.005
      sigma_factor: 0.0014247 #0.0014247

  object_transition:
    linear_sigma: 0.001 #0.0025
    angular_sigma: 0.002 #0.02
    velocity_factor: 0.8

  object_color:
   R: 102
   G: 178
   B: 255
